_includes/pbc/all/install-features/202304.0/install-the-product-offer-shipment-feature.md

```diff
@@ -253,11 +388,116 @@ class ConsoleDependencyProvider extends SprykerConsoleDependencyProvider
+|-----------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|
+| ShipmentTypeProductOfferPostCreatePlugin            | Persists product offer shipment type to persistence.                                                                           | Requires `ProductOfferTransfer.productOfferReference` to be set. Requires `ShipmentTypeTransfer.shipmentTypeUuid` to be set for each `ShipmentTypeTransfer` in `ProductOfferTransfer.shipmentTypes` collection. | Spryker\Zed\ProductOfferShipmentType\Communication\Plugins\ProductOffer                      |
+| ShipmentTypeProductOfferPostUpdatePlugin            | Deletes redundant product offer shipment types from Persistence. Persists missed product offer shipment types to Persistence.  | Requires `ProductOfferTransfer.productOfferReference` to be set. Requires `ShipmentTypeTransfer.shipmentTypeUuid` to be set for each `ShipmentTypeTransfer` in `ProductOfferTransfer.shipmentTypes` collection. | Spryker\Zed\ProductOfferShipmentType\Communication\Plugins\ProductOffer                      |
+| ShipmentTypeProductOfferExpanderPlugin              | Expands `ProductOfferTransfer` with related shipment types.                                                                    | Requires `ProductOfferTransfer.productOfferReference` to be set                                                                                                                                                 | Spryker\Zed\ProductOfferShipmentType\Communication\Plugins\ProductOffer                      |
+| ShipmentTypeProductOfferStorageExpanderPlugin       | Expands `ProductOfferStorageTransfer` expanded with shipment type storage data.                                                | Requires `ProductOfferStorageTransfer.productOfferReference` to be set.                                                                                                                                         | Spryker\Zed\ProductOfferShipmentTypeStorage\Communication\Plugin\ProductOfferStorage         |
+| MerchantProductOfferShipmentTypeStorageFilterPlugin | Filters out `ProductOfferShipmentTypeCollectionTransfer.productOfferShipmentTypes` with product offers with inactive merchants | Requires `ProductOfferShipmentTypeCollectionTransfer.productOfferShipmentTypes.productOffer` to be set.                                                                                                         | Spryker\Zed\MerchantProductOfferStorage\Communication\Plugin\ProductOfferShipmentTypeStorage |
```

Comment:
* We have to find the IG for a merchant feature or merchant offer feature and place it there.